src/classes/CW_AudienceCSVProcessHelperTest.cls

```diff
@@ -0,0 +1,145 @@
+/**
+* File Name   :	CW_AudienceCSVProcessHelperTest.cls
+* Description :	tests CW_AudienceCSVProcessHelper and CSVBatchProess
+*
+* Modification Log
+* ============================================================================
+* Ver Date	   Author		 Modification
+* --- ---------- -------------- --------------------------
+* 0.1 20/05/2014 Alex Panka     Initial version
+
+*
+*/
+@isTest
+private class CW_AudienceCSVProcessHelperTest {
+	
+	 public static String csvFile='Group Name,Account Name \nC2,1\nC2,2\nT3,3\nT3,4\nT4,5\nT4,6';
+	
+	
+	static testMethod void UploadBatchOnly()
+	{
+		//Nothibg to test just for build. Then we make rarl tests
+		CW_Campaign__c CampRec=TestDataUtil.getCampaign();
+		insert CampRec;
+		CW_Schedule__c sch = TestDataUtil.createSchedule(CampRec);
+		insert sch;
+		CW_CSV_Import_Status__c statRec = new CW_CSV_Import_Status__c(Marketing_Campaign__c= CampRec.ID);
+		insert statRec;
+		// Now import records - All a rejected s there no schedule/ rewards
+		//==============================================================
+		CSVBatchReader batchJobRead = new CSVBatchReader(csvFile, CampRec.Id,sch.Id);
+    	Test.StartTest();
+      	Database.executeBatch(batchJobRead);
+  		Test.StopTest();
+	}
+	
+
+	static testMethod void ProcessBatchOnly()
```

Comment:
* I suggest to annotate test methods with `@IsTest` for better readability.
* Methods should use the `lowerCamelCase` naming convention. I suggest rename accordingly.